integration_tests/src/tests/offline_processing.rs

```diff
@@ -123,7 +123,7 @@ pub fn offline_processing() -> Result<()> {
     if !(1.9..=2.1).contains(&duration) {
         return Err(anyhow!("Invalid duration: {}", duration));
     }
-    if !(930_000..=1_000_000).contains(&bit_rate) {
+    if !(930_000..=1_008_200).contains(&bit_rate) {
```

Comment:
- if something changed in the compositor that affected this test then adjust lower bounds too.
- if you are doing something like this make sure to communicate it clearly. You added that change after getting PR approved and I only noticed this by accident.